src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -212,6 +219,40 @@ function addEnrollErrorMessages(enrollErrorLines) {
     });
 }
 
+/**
+ * Triggers an AJAX request to retrieve the enroll status.
+ * Does the necessary post processing after the state of the AJAX request is returned.
+ */
+function triggerAndProcessAjaxSaveAction() {
+    getAjaxEnrollStatus()
+            .then((data) => {
+                if (data.statusMessagesToUser.length === 1
+                    && data.statusMessagesToUser[0].color === 'SUCCESS') {
```

Comment:
It's weird that you are deriving the status from the color instead of passing the status directly